src/app/about/about.component.spec.ts

```diff
@@ -68,7 +30,34 @@ describe('AboutComponent', () => {
     fixture.detectChanges();
   });
 
-  it('should create', () => {
-    expect(component).toBeTruthy();
+  it('should create an About Component', () => {
+    const aboutComponent = new AboutComponent();
+    expect(aboutComponent).toBeTruthy();
+  });
+
+  it('should have alt text property as brand', () => {
+    let compiled = fixture.debugElement.nativeElement;
+
+    let image: HTMLInputElement = compiled.querySelector('div.navbar-header img');
+    expect(image).toBeTruthy();
+    expect(image.alt).toBe('brand');
+  });
+
+  it('should have an footer navbar component', () => {
+    const footerNavbar = new FooterNavbarComponent();
+    expect(footerNavbar).toBeTruthy();
   });
+
+  it('should have an app-footer-navbar element', () => {
+    let compiled = fixture.debugElement.nativeElement;
+    expect(compiled.querySelector('app-footer-navbar')).toBeTruthy();
+  });
+
+  it('should have image-banner having mountains', () => {
```

Comment:
Is this test necessary? I mean what if we decide to change the layout, and not include an image, or decide to use it in another manner? Just wondering